cli/tests/integrations/test_package.py

```diff
@@ -703,3 +703,43 @@ def _package(name,
         yield
     finally:
         assert_command(['dcos', 'package', 'uninstall', name])
+
+
+def test_search_ends_with_wildcard():
```

Comment:
please add tests *before* the private functions.  private functions should be at the end of the file